### PR TITLE
add genrule for hack/update-bindata.sh

### PR DESCRIPTION
**Pull request by mikedanese**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/35373)

### CODE REVIEW

**Comment by sttts** [comment]
Running this with go 1.7.3 and bindata files looks good now.

**Comment by sttts** [comment]
But with 1.6 it gives different dependencies. I guess that's expected.

File: vendor/github.com/jteeuwen/go-bindata/LICENSE [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This work is subject to the CC0 1.0 Universal (CC0 1.0) Public Domain Dedication
+license. Its contents can be found at:
+http://creativecommons.org/publicdomain/zero/1.0
```
Comment (mikedanese) [inline vendor/github.com/jteeuwen/go-bindata/LICENSE]: update-godeps-licenses.sh refuses to pickup this license.

**Comment by spxtr** [comment]
So what's the plan here?

**Comment by mikedanese** [comment]
This is intended to fix the bindata problem not the go version instability. We should fix that separately. @spxtr PTAL

**Comment by spxtr** [comment]
Looks fine to me. @jayunit100 might be interested.

**Comment by spxtr** [comment]
@k8s-bot verify test this

**Comment by spxtr** [comment]
@eparis this adds a BUILD rule to `docs/user-guide` which is imported by go-bindata. We can look into why that import is necessary, but if it turns out to be necessary then we need this PR to go in anyway.

How do we make it so?

**Comment by mikedanese** [comment]
@spxtr there is nothing in docs/user-guide that could possibly be included (only markdown files). I'm just removing it from bindata search path.

**Comment by spxtr** [comment]
SGTM

**Pull request merged**